src/main/java/com/example/b2b_opportunities/Controller/PositionController.java

```diff
@@ -54,4 +56,14 @@ public PositionResponseDto editPosition(@PathVariable("id") Long id, @RequestBod
     public void deletePosition(@PathVariable("id") Long id, Authentication authentication) {
         positionService.deletePosition(id, authentication);
     }
+
+    @PatchMapping("/{id}/status")
```

Comment:
Why not just PutMapping? There's only one object anyway

Reply:
I also think that we should use PutMapping here